src/site/markdown/contributing.md

```diff
@@ -1,3 +1,21 @@
+# How to build
+
+#### Prerequisites
+
+* Maven 3.x
+* JDK 1.7 or later
+
+```
+$ cd hivemall
+
+# This is a workaround for resolving xgboost dependencies.
+$ mvn validate -Pxgboost
```

Comment:
Without `-Pxgboost`, we should face dependency error. And this profile is also set in [travis environment](https://github.com/myui/hivemall/blob/master/.travis.yml#L31).

```
Could not resolve dependencies for project io.github.myui:hivemall-xgboost:jar:0.4.2-rc.2: Could not find artifact ml.dmlc:xgboost4j:jar:0.10 in cloudera (https://repository.cloudera.com/artifactory/cloudera-repos/
```